Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:14-alpine
+
+ENV APP_HOME="/home/app/mcitmocks"
+
+RUN mkdir -p ${APP_HOME} && chown -R node:node ${APP_HOME}
+
+WORKDIR ${APP_HOME}
+
+COPY package*.json ./
+RUN yarn
+
+COPY . . 
```

Comment:
It seems like this might be making a copy of the whole repo folder into the image? That might explain why my local changes in the folder aren't showing up in what gets served

Reply:
If true, it might also explain why you couldn't get hot reload working